lib/commands/skill/add-locales/helper.js

```diff
@@ -0,0 +1,130 @@
+const fs = require('fs-extra');
+const path = require('path');
+const R = require('ramda');
+const async = require('async');
+
+const httpClient = require('@src/clients/http-client');
+const SkillMetadataController = require('@src/controllers/skill-metadata-controller');
+const CliError = require('@src/exceptions/cli-error');
+const ResourcesConfig = require('@src/model/resources-config');
+const Manifest = require('@src/model/manifest');
+const stringUtils = require('@src/utils/string-utils');
+const CONSTANTS = require('@src/utils/constants');
+
+module.exports = {
+    initiateModels,
+    addLocales
+};
+
+/**
+ * Start with loading the project Models global instances.
+ * @param {String} profile
+ */
+function initiateModels(profile) {
+    new ResourcesConfig(path.join(process.cwd(), CONSTANTS.FILE_PATH.ASK_RESOURCES_JSON_CONFIG));
+    const skillPackageSrc = ResourcesConfig.getInstance().getSkillMetaSrc(profile);
+    if (!stringUtils.isNonBlankString(skillPackageSrc)) {
+        throw new CliError('Skill package src is not found in ask-resources.json.');
```

Comment:
I think if we want to have those validation they should be done inside if the model. For example, getSkillMetaSrc should throw error. Otherwise we have duplicate validations everywhere.

Reply:
It feels to me the definition of our Model module should be more explicitly defined in two part, one is just for JSON definition with centralized business logic, another will be to serve all the file operation, where the ensure of existing or some other validation can happen in that layer. We need a one pager for this to arrange the interfaces, not for this task